src/main/java/org/sagebionetworks/web/client/PortalGinInjector.java

```diff
@@ -535,7 +535,7 @@ public interface PortalGinInjector extends Ginjector {
 	ColumnModelTableRowEditorWidget createColumnModelEditorWidget();
 
 	TableEntityWidget createNewTableEntityWidget();
-	TableEntityPlotsWidget createNewTableEntityPlotsWidget();
+	TableEntityWidgetV2 createNewTableEntityWidgetV2();

```

Comment:
Renamed from TableEntityPlotsWidget to TableEntityWidgetV2 (since it may not show plots!)